package.json

```diff
@@ -36,7 +36,7 @@
     "locktt": "dist/index.js"
   },
   "engines": {
-    "node": "^12.13.0 || ^14.15.0 || ^16.10.0 || >=17.0.0"
+    "node": "^12.13.0 || ^14.15.0 || ^16.10.0 || ^16.2.0 || >=17.0.0"
```

Comment:
```suggestion
    "node": ">=12.13.0"
```

Reply:
Interesting this could work. I think only `jest` has some node engine dependencies but other packages won't complain from what I have seen. Since `jest` is a dev dependency I think this should be fine

Reply:
Great